test/alternator/test_lsi.py

```diff
@@ -242,7 +242,7 @@ def test_lsi_describe(test_table_lsi_4):
     assert(sorted([lsi['IndexName'] for lsi in lsis]) == ['hello_x1', 'hello_x2', 'hello_x3', 'hello_x4'])
     for lsi in lsis:
         assert lsi['IndexArn'] == desc['Table']['TableArn'] + '/index/' + lsi['IndexName']
-    assert lsi['Projection'] == {'ProjectionType': 'ALL'}
+        assert lsi['Projection'] == {'ProjectionType': 'ALL'}
```

Comment:
Good catch. I actually noticed this yesterday too, and wanted to send a patch and forgot :-)
You're right that the test was just testing the last lsi, which is better than nothing, but definitely not what was intended.